packages/@okta/vuepress-theme-default/assets/css/okta/components/Breadcrumbs.scss

```diff
@@ -1,15 +1,38 @@
 @import '../icons/functions';
 @import '../icons/variables-unicode';
 
-.PageContent-main ol.breadcrumb {
+.breadcrumb-nav {
+	border-bottom: 2px solid #e5e9eb;
+
+	@include media('>=1024px') {
+		margin-top: 95px;
+	}
+	@include media('<1024px') {
+		margin-top: 25px;
+	}
+	@include media('<860px') {
```

Comment:
Perhaps a comment to indicate why this isn't a logical "smaller is smaller" flow for future devs?

Reply:
If I knew why this was, I would make a note.... :)